app/services/api/map_hesa_attributes/v01.rb

```diff
@@ -1,28 +1,14 @@
 # frozen_string_literal: true
 
 module Api
-  module Hesa
-    class MapTraineeAttributes
+  module MapHesaAttributes
+    class V01
       include ServicePattern
       include HasDiversityAttributes
       include HasCourseAttributes
 
       ATTRIBUTES = %i[
-        first_names
-        last_name
-        email
-        date_of_birth
-        ethnic_background
-        sex
-        course_subject_one
-        course_subject_two
-        course_subject_three
-        itt_end_date
-        study_mode
-        training_route
         nationality
```

Comment:
This provides a way to let vendors use arbitrary key names similar to the current hesa creation process and allows us to map those to the keys we expect. Eg in `CreateFromHesa` `nationality` will come in as `'GB'`. We can allow vendors to still use this format.